data/explorations/all_interactions/all_interactions.yaml

```diff
@@ -816,62 +816,27 @@ states:
     classifier_model_id: null
     content:
       content_id: content
-      html: <p>Math Expression Input. Provide a general equation of a 2D line in a
-        2D space. Let y be an independent variable which depends on the dependent
-        variable x.</p>
+      html: <p>Math Expression Input. Provide a formula for calculating the area
```

Comment:
It's 'provide the formula', right? Since, you are looking for only pi * r * r and not anything else?

Reply:
Ah yes! done.